<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConfigurationWizard</class>
 <widget class="QWizard" name="CConfigurationWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swift configuration wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <widget class="QWizardPage" name="wp_Simulator">
   <property name="title">
    <string>Your simulator</string>
   </property>
   <property name="subTitle">
    <string>The simulator(s) you are using</string>
   </property>
   <layout class="QVBoxLayout" name="vl_Simulators">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="fr_Simulator">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gl_Simulators">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="cb_P3D">
         <property name="text">
          <string>P3D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="cb_FSX">
         <property name="text">
          <string>FSX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="cb_FS9">
         <property name="text">
          <string>FS9</string>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="cb_XP">
         <property name="text">
          <string>XPlane</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_P3D">
         <property name="text">
          <string>&lt;img width=&quot;100&quot; src=&quot;:/simulators/icons/simulators/Prepar3D.png&quot;/&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbl_FSX">
         <property name="text">
          <string>&lt;img width=&quot;100&quot; src=&quot;:/simulators/icons/simulators/FSX.png&quot;/&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lbl_XP">
         <property name="text">
          <string>&lt;img width=&quot;100&quot; src=&quot;:/simulators/icons/simulators/XPlane.png&quot;/&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbl_FS9">
         <property name="text">
          <string>&lt;img width=&quot;100&quot; src=&quot;:/simulators/icons/simulators/FS9.png&quot;/&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wp_CopySettings">
   <property name="title">
    <string>Copy configuration from another installation</string>
   </property>
   <property name="subTitle">
    <string>Copy settings</string>
   </property>
   <layout class="QVBoxLayout" name="vl_CopySettings">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CCopyConfigurationComponent" name="comp_CopySettings">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wp_CopyCaches">
   <property name="title">
    <string>Copy configuration from another installation</string>
   </property>
   <property name="subTitle">
    <string>Cache data</string>
   </property>
   <layout class="QVBoxLayout" name="vl_CopyCaches">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CCopyConfigurationComponent" name="comp_CopyCaches">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CCopyConfigurationComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/copyconfigurationcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
